src/components/Books.js

```diff
@@ -0,0 +1,74 @@
+import { useSelector, useDispatch } from 'react-redux';
+import { useEffect } from 'react';
+import { addBook, fetchBooks } from '../redux/books/booksSlice';
+import Book from './Book';
+
+const Books = () => {
+  const { books, isLoading } = useSelector((state) => state.books);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchBooks());
+  }, [dispatch]);
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const title = document.getElementById('titleInput').value;
+    const author = document.getElementById('authorInput').value;
+    const category = 'non-fiction';
+    const id = Math.random().toString(36).substr(2, 9);
+    const book = {
+      item_id: id, title, author, category,
+    };
+    dispatch(addBook(book));
+  };
+
+  if (isLoading) {
+    return <h2>Loading...</h2>;
+  }
+
+  return (
+    <div>
+      <ul className="books">
+        {
+          books.map((book) => (
+            <li key={book.item_id}>
+              <Book item={book} />
+            </li>
+          ))
+        }
+      </ul>
+      <div className="Line" />
+      <div className="form-div">
+        <h2 className="Title">ADD NEW BOOK</h2>
+        <form onSubmit={(e) => handleSubmit(e)}>
+          <label htmlFor="titleInput">
+            <input
+              className="Lesson-Panel"
+              type="text"
+              id="titleInput"
+              name="title"
+              placeholder="Title"
+            />
+          </label>
+          <br />
+          <label htmlFor="authorInput">
+            <input
+              className="Lesson-Panel"
+              type="text"
+              id="authorInput"
+              name="author"
+              placeholder="Author"
+            />
+          </label>
+          <br />
+          <button type="submit" className="ADD-BOOK">
+            Add Book
+          </button>
+        </form>
+      </div>
```

Comment:
- [ ] @gaks1 Great work on your design however the input section width is bigger than other elements, which makes it not match the zeplin design given kindly reduce the width to match the other elements' width.
![image](https://user-images.githubusercontent.com/61408860/230618733-8f6b5146-4a82-4094-897a-c832d89281ce.png)

Reply:
- [ ] @gaks1 Great work on your design, however, your button width is smaller than the design given. also, the author's width is bigger than in the design, check the comparison below

Zeplin Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/230619286-ee25fa8c-e6b9-4889-8d8a-3a893a641c46.png) | ![image](https://user-images.githubusercontent.com/61408860/230619427-d0505d53-2552-4554-b933-e13eb31bffd1.png)